development/docker-compose.yml

```diff
@@ -204,6 +207,7 @@ services:
       INFRAHUB_WORKFLOW_PORT: "${INFRAHUB_WORKFLOW_PORT:-4200}"
       PREFECT_API_URL: "http://task-manager:4200/api"
     volumes:
+      - "./infrahub.toml:/source/development/infrahub.toml"
```

Comment:
I think this one could do more harm than good for us because if the format of the file changes in develop, it won't be compatible with the demo environment (stable image)

if I understand the root cause of the issue, we need to ensure that we can't run `demo.build`